ateam_kenobi/src/kenobi_node.cpp

```diff
@@ -218,11 +222,21 @@ class KenobiNode : public rclcpp::Node
   {
     world_.current_time = std::chrono::steady_clock::now();
     world_.referee_info.running_command = game_controller_listener_.GetGameCommand();
+    world_.referee_info.command_time =
+      std::chrono::system_clock::time_point(
+      std::chrono::nanoseconds(
+        rclcpp::Time(
+          game_controller_listener_.GetLatestRefereeMessage().command_timestamp).nanoseconds()));
+    world_.referee_info.prev_command = game_controller_listener_.GetPreviousGameCommand();
     world_.referee_info.current_game_stage = game_controller_listener_.GetGameStage();
     if (game_controller_listener_.GetOurGoalieID().has_value()) {
       world_.referee_info.our_goalie_id = game_controller_listener_.GetOurGoalieID().value();
     }
+    if (game_controller_listener_.GetTheirGoalieID().has_value()) {
+      world_.referee_info.their_goalie_id = game_controller_listener_.GetTheirGoalieID().value();
+    }
     in_play_eval_.Update(world_);
+    double_touch_eval_.update(world_);
```

Comment:
We should probably pick some more specific style guidelines beyond our automated tests/linter at some point. It's probably unideal to have some  classes with an `update` and others with an `Update` method...

Reply:
I think that only does formatting, which we already have. I don't think clang-format can enforce symbol names.

Reply:
Darn